modules/logging-bucket/README.md

```diff
@@ -20,6 +20,23 @@ module "bucket" {
 # tftest modules=1 resources=1 inventory=project.yaml
 ```
 
+### Create custom logging bucket in a project enabling Log Analytics and dataset link
+
+```hcl
+module "bucket" {
+  source      = "./fabric/modules/logging-bucket"
+  parent_type = "project"
+  parent      = var.project_id
+  id          = "mybucket"
+  log_analytics = {
+    enable = true
+    linked_dataset = {
+      link_id = "log"
+    }
+  }
+}
+# tftest modules=1 resources=2 inventory=log_analytics.yaml
```

Comment:
Can you extend the example to include the creation of the dataset with the bigquery module?

Reply:
The dataset is created automatically by the resource. No need to create id in advance, actually it fails if you pass an existing dataset.